code/modules/power/tesla/energy_ball.dm

```diff
@@ -129,6 +129,10 @@
 /obj/singularity/energy_ball/Bumped(atom/movable/AM)
 	dust_mobs(AM)
 
+/obj/singularity/energy_ball/attack_tk(mob/user)
+	to_chat(user, "<span class='userdanger'>That was a shockingly dumb idea.</span>")
+	explosion(user.loc,2,2,4,flame_range = 2)
```

Comment:
this should make a massive zap centered on the TKer

Reply:
I think he means something like when you throw something at an electrified grill

Reply:
That's actually what I did at first, but getting close to the tesla without dying already requires full shock protection so it just ended up being kinda lame

Reply:
IMO it should fry the brain, deleting it with no external signs except a puff of smoke